sensing/pointcloud_preprocessor/schema/blockage_diag.schema.json

```diff
@@ -0,0 +1,113 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameter for blockage diag filter",
+  "type": "object",
+  "definitions": {
+    "blockage_diag": {
+      "type": "object",
+      "properties": {
```

Comment:
parameter "horizontal_ring_id" is missed. 
Please also updated in the paramter file.